### DISCUSSION TITLE
How to display variants $currency on products query

### DISCUSSION THREAD

**Discussion by jaymar-webriq** [swellstores/community#288] [Help] [answered]
Hello I have here a sample on how to display multi currency on `/products:variants` which is 
```
const query_params = {"$currency": ["USD","CAD"]}
await swell.get('/products:variants/<id>', query_params);
```

now i want to expand it on `/products` query, this is my query that is not working
```
await swell.get('/products', {
  "expand": ["variants:0"],
  "$currency": ["USD","CAD"]
});
```

variants results dont have $currency field on it
thanks

**Comment by sags95** [comment]
Hey @jaymar-webriq, I'll file a bug for the team so we can investigate wether this is a bug or a gap.

**Reply by jaymar-webriq** [reply]
thank you @sags95

**Reply by steven-k1** [reply, marked as answer]
@jaymar-webriq A fix for this was deployed and should be working as expected now.
